tests/csi-sanity/ga-docker-compose1.yaml

```diff
@@ -0,0 +1,31 @@
+version: '3'
```

Comment:
suggest to rename docker-compose-nosnapshotcaps.yaml
Why do you have ga- prefix?

Reply:
will rename